### PR TITLE
Issue #9412: updated example of AST for TokenTypes.CTOR_CALL

### LINKED ISSUE #9412: update example of AST for TokenTypes.CTOR_CALL
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CTOR_CALL

Needs to be extended to show example of AST in format that printed by our CLI (change "+-" to "`-") .
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by AbhinavRajputEXE**
Fixes #9412 : update example of AST for TokenTypes.CTOR_CALL

### CODE REVIEW

**Comment by AbhinavRajputEXE** [comment]
@romani , please review this PR

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -1441,16 +1441,13 @@
...
-     *             +--NUM_INT (1)
-     *     +--RPAREN ())
-     *     +--SEMI (;)
+     * CTOR_CALL -&gt; this
+     *  |--LPAREN -&gt; (
+     *  |--ELIST -&gt; ELIST
+     *      |--EXPR -&gt; EXPR
+     *          |--NUM_INT -&gt; 1
+     *  |--RPAREN -&gt; )
+     *  |--SEMI -&gt; ;
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1450]: Please share output of cli for such code, I think this line should have different pseudo graphic element

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java (hunk 1 above)
Comment (AbhinavRajputEXE) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1450]: Screenshot (535)
Should I make it look like the AST above?
in the SUPER_CTOR_CALL example?

Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1450]: Print whole ast by cli.

Comment (AbhinavRajputEXE) [reply src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java]: @romani , I changed it, this time I created a fresh AST using CLI.
Screenshot (539)

**Comment by romani** [comment]
Squash commits merged in one

**Comment by AbhinavRajputEXE** [comment]
> Squash commits merged in one

did it.

**Review by romani — APPROVED** [review]

**Pull request merged**